_posts/2017-10-17-first-haskell-class-meeting.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "First Haskell class meeting"
```

Comment:
`Haskell class` sounds like someone comes and teaches us. What we have is more like learning group, even though some people are far beyond from being beginners